src/middleware/parser.js

```diff
@@ -33,11 +35,11 @@ const parseIncomingBody = async (ctx, inputFormat, next) => {
     }
 
     const options = {
-      limit: '1mb',
+      limit: config.parser.limit,
       xmlOptions: {
-        trim: true,
-        explicitRoot: false,
-        explicitArray: false
+        trim: (config.parser.xmlOptions.trim == 'true'),
```

Comment:
Minor but for consistency, this logic can be put in the config file

Reply:
done in b95e138